### DISCUSSION TITLE
带初始化的二维数组，如何处理分层的结构

### DISCUSSION THREAD

**Discussion by zhengzhch** [arcsysu/SYsU-lang#31] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/56723398/163353468-7a185ce2-92ac-4e7e-b2aa-72f5d46e34ea.png)
![image](https://user-images.githubusercontent.com/56723398/163353513-6b22bac1-f055-495a-bdd2-7a9360257d62.png)
二维数组初始化，后面的初始化需要根据二维数组的结构来分层，有没有一个比较好的处理方法

**Comment by Yorkking** [comment, marked as answer]
评测脚本中已经不要求识别多维（含一维）数组初始化的列表内容了。你只需要返回一个 `{"kind": "InitListExpr", "inner": []}` 即可。
